static/js/stream_data.js

```diff
@@ -1,7 +1,7 @@
 var stream_data = (function () {
 
 var exports = {};
-
+zrequire('marked', 'third/marked/lib/marked');
```

Comment:
Is this necessary? We use `zrequire` for testing purposes. You should add this line to the test file if you are getting errors.

Reply:
@lonerz i tried the `global.marked` does not work as i said at zulipchat. maybe should use `markdown` method?